virttest/qemu_vm.py

```diff
@@ -1897,6 +1900,48 @@ def sort_key(dev):
                               pci_bus=pci_bus)
                 iov += 1
 
+        def get_cid(cid):
+            """ Get an unused guest cid from system """
+            while cid:
+                cid_c = struct.pack('L', cid)
+                try:
+                    fcntl.ioctl(
+                        vsock_fd, arch.VHOST_VSOCK_SET_GUEST_CID, cid_c)
+                except IOError as e:
+                    if e.errno == errno.EADDRINUSE:
```

Comment:
I prefer to add a comment here, to explain the given cid already in use. Not must, it is up to you.

Reply:
@yanan-fu I think this process should be silence instead of verbose, since this is generated automatically, not set by user, and user actually do not need care about the cid.